esp/esp/program/forms.py

```diff
@@ -62,23 +62,46 @@ class ProgramCreationForm(BetterModelForm):
+                                                        'Do you want a pre-program quiz for teachers?': [x.id for x in ProgramModule.objects.filter(admin_title='Teacher Logistics Quiz')],
+                                                        'Will you have any additional non-survey forms that teachers should fill out?': [x.id for x in ProgramModule.objects.filter(admin_title='Teacher Custom Form')],
+                                                        'Will you have any (non-survey) forms that students should fill out?': [x.id for x in ProgramModule.objects.filter(admin_title='Student Custom Form')],
+                                                        'Will you have more than one lunch period (per day)?': [x.id for x in ProgramModule.objects.filter(admin_title='Student Lunch Period Selection')],
+                                                        'Would you be willing to solicit donations for LU?': [x.id for x in ProgramModule.objects.filter(admin_title='Donation Module')],
+                                                        'Do you plan to have teacher training or interviews?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Teacher Training and Interview Signups', 'Manage Teacher Training and Interviews'])],
+                                                        mark_safe('Will you use lottery admission (as opposed to first come, first served) for <b>classes</b>?'): [x.id for x in ProgramModule.objects.filter(admin_title__in=['Two-Phase Student Registration', 'Lottery Frontend'])],
+                                                        mark_safe('Will you use lottery registration (as opposed to first come, first served) to the <b>program</b>?'): [x.id for x in ProgramModule.objects.filter(admin_title__in=['Student Registration Phase Zero', 'Manage Student Registration Phase Zero'])],
+                                                        'Do students have to apply to individual classes?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Application Review for Admin', 'Admin Admissions Dashboard'])],
+                                                        'If yes, can teachers admit them (as opposed to just admins)?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Teacher Admissions Dashboard', 'Application Reviews for Teachers', 'Application Review for Admin', 'Admin Admissions Dashboard'])]
+                                                       })
+        # Include additional or new modules that haven't been added to the list
+        for x in ProgramModule.objects.filter(choosable=0):
+            if x.id not in sum(self.program_module_question_ids.values(), []): # flatten list of modules
+                self.program_module_question_ids['Would you like to include the {} module?'.format(x.admin_title)] = [x.id]
```

Comment:
Are people expected to add relevant questions when they make their modules so ideally they dont just plop at the bottom of the list, and if so, is there appropriate dev documentation about adding the new module to this framework?

Reply:
I expect most new modules will be included by default and won't need a question, but if there are new ones that get in the way of people who don't use them, then yes, the author should add a new question. I will add a note in the program module dev docs. Thanks

Reply:
added in 655eaa1